includes/scheduler/class-statistics.php

```diff
@@ -0,0 +1,259 @@
+<?php
+/**
+ * Statistics scheduler class file.
+ *
+ * Handles scheduled collection of ActivityPub statistics.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Scheduler;
+
+use Activitypub\Mailer;
+use Activitypub\Statistics as Statistics_Collector;
+
+/**
+ * Statistics scheduler class.
+ */
+class Statistics {
+
+	/**
+	 * Initialize the class, registering WordPress hooks.
+	 */
+	public static function init() {
+		\add_action( 'activitypub_collect_monthly_stats', array( self::class, 'collect_all_monthly_stats' ) );
+		\add_action( 'activitypub_compile_annual_stats', array( self::class, 'compile_and_send_annual_stats' ) );
+		\add_filter( 'cron_schedules', array( self::class, 'add_cron_schedules' ) );
+	}
+
+	/**
+	 * Add custom cron schedules.
+	 *
+	 * @param array $schedules Existing cron schedules.
+	 *
+	 * @return array Modified cron schedules.
+	 */
+	public static function add_cron_schedules( $schedules ) {
```

Comment:
`add_cron_schedules`, `register_schedules`, `deregister_schedules` should live in `Scheduler`.